frontend/src/components/CodeEditorInstructions.tsx

```
@@ -1,19 +1,42 @@
         <div className="guideContainer">
-            <h3>Why We Don't Have an Embedded Code Editor</h3>
-            <p>
-                Our focus is on preparing you for real technical interviews,
-                where you'll typically use either an online coding platform like
-                LeetCode or an IDE like VSCode. By practicing in these
-                environments, you'll build familiarity with the tools you'll
-                encounter during interviews. This approach ensures you're
-                comfortable with industry-standard coding environments, not a
-                custom editor you won't see in actual interviews.
-            </p>
+            <div className={styles.title} onClick={toggleVisibility}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)